Annotato/Annotato/ViewController.swift

```diff
@@ -0,0 +1,181 @@
+//
+//  ViewController.swift
+//  Annotato
+//
+//  Created by Sivayogasubramanian on 10/3/22.
+//
+
+import UIKit
+import PDFKit
+import UniformTypeIdentifiers
+
+class ViewController: UIViewController {
+    let pdfView = PDFView()
+    var pdfTapGestures: [UITapGestureRecognizer] = []
+    var tripleTapAnnotateGesture = UITapGestureRecognizer()
+    @IBOutlet private var nextButton: UIBarButtonItem!
+    @IBOutlet private var importFilesButton: UIBarButtonItem!
+
+    let buttonHeight = 50.0
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addPdfView()
+        addObservers()
+        addTripleTapGestureAddAnnotation()
+        initializeGoToDocumentViewButton()
+    }
+
+    private func initializeGoToDocumentViewButton() {
+        let documentViewButton = FilledButton(
+            title: "Doc List",
+            frame: CGRect(x: .zero, y: .zero, width: 100, height: 100)
+        )
+        documentViewButton.titleLabel?.adjustsFontSizeToFitWidth = true
+
+        documentViewButton.center = .init(x: 100, y: 100)
+        documentViewButton.addTarget(self, action: #selector(didTapDocumentButton), for: .touchUpInside)
+        view.addSubview(documentViewButton)
+    }
+
+    private func addPdfView() {
+        view.addSubview(pdfView)
+        guard let url = Bundle.main.url(forResource: "Lab01Qns", withExtension: "pdf") else {
+            print("No such pdf")
```

Comment:
Can we remove all the print statements? We can either log errors and states (I think siva set up a logger). Or we can present an alert if users need to know something.